tests/integration/test_security_report.py

```diff
@@ -10,6 +10,46 @@
 HPKP_IGNORED_FIELDS = ("event_id",)
 
 
+def id_fun1(origins):
+    allowed, should_be_allowed = origins
+    should_it = "" if should_be_allowed else "not"
+    return f"{str(allowed)} should {should_it} be allowed "
+
+
+@pytest.mark.parametrize(
+    "allowed_origins",
+    [(["*"], True), (["valid.com"], True), ([], False), (["invalid.com"], False)],
+    ids=id_fun1,
```

Comment:
These tests seem duplicate to the ones in `test_basic.py`. Can we reduce them to just one or two tests that ensure basic functionality? We don't need to test all permutations here.

Reply:
`test_basic.py` tests the store endpoint, this test checks the security endpoint for all the types of security reports ( csp, expect_ct, expect_staple and hpkp)... I'll trim it down to a positive and an negative test for each report type ...

Reply:
That's right, but the origin logic is 100% shared. It is even the exact same code path for security reports and regular events. Given this, doesn't it suffice to just test one of them?

Reply:
Ok, removed everything except one test for csp valid/invalid